inputfiles/addedTypes.json

```diff
@@ -93,6 +93,18 @@
                             "name": "change",
                             "type": "Event"
                         },
+                        {
+                            "name": "compositionstart",
```

Comment:
Since there is no corresponding `oncompositionstart` etc., these should go to `attributeless-events` instead. Also, please run `npm i && npm run build && npm run baseline-accept` and commit the results.

Reply:
Thanks for the pointer! I've updated the patch.